### PR TITLE
.spacemacs template docstring revision

### PR DESCRIPTION
**Pull request by wygulmage**
I edited the docstrings of the main functions of the .spacemacs template for grammar and brevity. It makes more sense to me, so I figured I'd share it.

### CODE REVIEW

File: core/templates/.spacemacs.template [hunk 1]
```diff
@@ -69,11 +68,9 @@ values."
...
 
 (defun dotspacemacs/init ()
-  "Initialization function.
-This function is called at the very startup of Spacemacs initialization
-before layers configuration.
-You should not put any user code in there besides modifying the variable
-values."
+  "Initialization function for Spacemacs settings:
+This function is called at the beginning of Spacemacs initialization, before
+layers configuration. It should only modify the values of Spacemacs settings."
```
Comment (robbyoconnor) [inline core/templates/.spacemacs.template]: I don't see an issue with the old version.

File: core/templates/.spacemacs.template [hunk 2]
```diff
@@ -3,9 +3,8 @@
 ;; It must be stored in your home directory.
 
 (defun dotspacemacs/layers ()
-  "Configuration Layers declaration.
-You should not put any user code in this function besides modifying the variable
-values."
+  "Configuration layers:
+This function should only modify layer settings."
```
Comment (robbyoconnor) [inline core/templates/.spacemacs.template]: No issue here.

File: core/templates/.spacemacs.template [hunk 3]
```diff
@@ -326,22 +323,19 @@ values."
...
-  "Initialization function for user code.
-It is called immediately after `dotspacemacs/init', before layer configuration
-executes.
- This function is mostly useful for variables that need to be set
-before packages are loaded. If you are unsure, you should try in setting them in
+  "Initialization function for user code:
+This function is called immediately after `dotspacemacs/init', before layer
+configuration. It is mostly useful for variables that should be set before
+packages are loaded. If you are unsure, try setting them in
 `dotspacemacs/user-config' first."
```
Comment (robbyoconnor) [inline core/templates/.spacemacs.template]: :white_check_mark:

File: core/templates/.spacemacs.template [hunk 4]
```diff
@@ -326,22 +323,19 @@ values."
...
-layers configuration.
-This is the place where most of your configurations should be done. Unless it is
-explicitly specified that a variable should be set before a package is loaded,
-you should place your code here."
-  )
+  "Configuration function for user code:
+This function is called at the very end of Spacemacs initialization, after
+layers configuration. Put your configuration code here, except for variables
+that should be set before packages are loaded."
+)
```
Comment (robbyoconnor) [inline core/templates/.spacemacs.template]: :white_check_mark:

**Comment by robbyoconnor** [comment]
Please actually title your PR...

**Comment by wygulmage** [comment]
Will do. Sorry that you had to deal with my mistakes, but I do appreciate the corrections. I haven't used Github much, so I'm feeling around in the dark.
—Keith

**Comment by robbyoconnor** [comment]
It's okay!

**Pull request closed without merging**